Framework/Algorithms/src/IQTransform.cpp

```diff
@@ -118,8 +117,10 @@ void IQTransform::exec() {
   outputWS->setYUnit("");
   // Copy the data over. Assume single spectrum input (output will be).
   outputWS->setPoints(0, tmpWS->points(0));
-  MantidVec &Y = outputWS->dataY(0) = tmpWS->dataY(0);
-  outputWS->dataE(0) = tmpWS->dataE(0);
+  outputWS->setSharedY(0, tmpWS->sharedY(0));
+  outputWS->setSharedE(0, tmpWS->sharedE(0));
+
+  auto &Y = outputWS->mutableY(0);
```

Comment:
You are triggering a copy even in the cases where the background is not subtracted. Maybe simply use

```cpp
if (backround > 0.0)
  outputWS->mutableY(0) -= background;
```

below.

(Actually this is really minor here, since we are simply dealing with a single spectrum, but I looks slightly more readable, maybe?)